src/libs/BouncerProvider.js

```diff
@@ -47,6 +47,14 @@ export default class BouncerProvider {
         this.bnc.path = path || '';
         this.bnc.enabled = true;
 
+        const bncNetwork = this.state.networks.find((network) => network.is_bnc);
```

Comment:
This could do with a quick comment explaining why it's looking for a is_bnc network and why it's reading the password field